protocol/common/raw_message.go

```diff
@@ -45,6 +45,15 @@ const (
 	ResendMethodSendCommunityMessage ResendMethod = 2
 )
 
+// MessagePriority determines the ordering for publishing  message
+type MessagePriority = int
+
+var (
+	LowPriority    MessagePriority = 0
+	MediumPriority MessagePriority = 1
+	HighPriority   MessagePriority = 2
```

Comment:
It seems that low/high priority could be enough? I can see you don't use `MediumPriority`

Reply:
MediumPriority is the default now in https://github.com/waku-org/go-waku/pull/1171  
It should take into account any message I missed to set an specific priority